config/routes.rb

```diff
@@ -3,4 +3,9 @@
 
   devise_for :users
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  namespace :api do
+    namespace :v1 do
+      resources :wizards, only: [:index, :show]
```

Comment:
If this PR is for `index` alone (which is great scoping), I'd change this to only have `index`